packages/nbformat/package.json

```
@@ -17,6 +17,7 @@
   ],
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
+  "type": "module",
```

```suggestion
  "type": "module",
  "main": "lib/index.js",
  "types": "lib/index.d.ts",
```

To make the linter happy.